lib/components/molecules/AtomicCarousel/AtomicCarousel.js

```diff
@@ -0,0 +1,341 @@
+    return (
+      <div className="carousel-container">
+        {data.length !== 0 && (
+          <>
+            {navigation && (
+              <div className="action previous">
+                <Button
+                  data-action="prev"
+                  className={classnames({
+                    'prev-icon': prevIcon,
+                  })}
+                  onClick={handlePrev}
+                  disabled={currentSlide === 1}
+                  onKeyUp={handlePreviousKey}
+                  title={prevLabel}
+                  type="button"
+                >
+                  <Icon path={prevIcon} alt="previous" title="previous" />
+                </Button>
+              </div>
+            )}
+            <div
+              className={classnames('carousel-slides', {
+                fadeIn: fade && isAnimationRunning,
+              })}
+              onMouseEnter={handleMouseEnter}
+              onMouseLeave={handleMouseLeave}
+            >
+              {getPerFrameItems(data, perSlideFrame, currentSlide)}
+            </div>
+            {navigation && (
+              <div className="action next">
+                <Button
+                  className={classnames({ 'next-icon': nextIcon })}
+                  data-action="next"
+                  title={nextLabel}
+                  onClick={handleNext}
+                  disabled={slideCount === currentSlide}
+                  onKeyUp={handleNextKey}
+                  type="button"
+                >
+                  <Icon path={nextIcon} alt="next" title="next" />
+                </Button>
+              </div>
+            )}
+          </>
+        )}
+      </div>
+    );
+  };
+
+  const getHeader = () => {
+    return (
+      <header id="carousel">
+        <Heading {...headers} className="carousel-heading">
+          {heading}
+        </Heading>
+      </header>
+    );
+  };
+
+  const ariaLiveUpdate = () => {
+    return (
+      <div className="hidden" aria-live="polite" aria-atomic="true" tabIndex="-1">
+        {`Item ${currentSlide} of ${slideCount}`}
+      </div>
+    );
+  };
+
+  const getAnimationPanel = () => {
+    return (
+      <div className="animation-panel" aria-label="animation">
+        <Button type="button" onClick={handleAnimation}>
+          <Icon
+            path={
+              (playIcon && !isAnimationRunning ? playIcon : false) ||
+              (pauseIcon && isAnimationRunning ? pauseIcon : false)
+            }
+            alt="next"
+            title="next"
+          />
+        </Button>
+      </div>
+    );
+  };
+
+  return (
+    <>
+      <section className={classnames('carousel', className)} aria-labelledby="carousel">
+        {displayHeader && getHeader()}
+        {getSliderSection()}
+        {dots && getList(setCurrentSlide, currentSlide)}
+        {getAnimationPanel()}
+        {ariaLiveUpdate()}
+      </section>
+    </>
+  );
+};
+
+AtomicCarousel.defaultProps = {
+  dots: true,
```

Comment:
Why is every prop have a default value? In file lib/components/molecules/AtomicCarousel/types/index.js most of the props are mandatory, Mandatory props will always should be passed by user only optional props can have a default value.